src/Symfony/Component/Clock/MockClock.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Clock;
+
+/**
+ * A clock that always returns the same date, suitable for testing time-sensitive logic.
+ *
+ * @author Nicolas Grekas <p@tchwork.com>
+ */
+final class MockClock implements TimeInterface
+{
+    private \DateTimeImmutable $now;
+
+    public function __construct(\DateTimeImmutable|\DateTimeZone|string $now = null)
+    {
+        if (\is_string($now ??= 'UTC')) {
+            try {
+                $now = new \DateTimeZone($now);
+            } catch (\Exception) {
+                $now = new \DateTimeImmutable($now);
+            }
```

Comment:
```suggestion
            $now = new \DateTimeImmutable($now);
```

A timezone identifier is a valid input for `DateTimeImmutable`'s constructor.

Reply:
> A timezone identifier is valid input for DateTimeImmutable's constructor.

Wow, I didn't know that :)
PR updated to handle two arguments.